polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
```

Comment:
I did not consider it critical, but since we are debating "in-place" vs. "in place", I will add that "Does not return a copy" is incomplete. It should be given a subject, changed to the passive voice (e.g., "A copy is not returned."), or deleted because it is redundant with the previous sentence.